dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -13,21 +13,17 @@ def update_last_confirmation_date
   def update
     return unless school_info_params[:school_id].present? || school_info_params[:country].present?
 
-    new_school_info_params =
-      if school_info_params[:full_address]&.blank?
-        school_info_params.except(:full_address)
-      else
-        school_info_params
-      end
-
-    if new_school_info_params[:country]&.downcase&.eql? 'united states'
-      new_school_info_params[:country] = 'US'
-    end
+    # school_info_params.delete(:full_address) if school_info_params[:full_address]&.blank?
+    e = school_info_params.merge({country: 'US'})
+    e.delete(:full_address) if e[:full_address]&.blank?
+    school_info_params = e
 
     existing_school_info = current_user.last_complete_school_info
-    existing_school_info&.assign_attributes new_school_info_params
+    existing_school_info&.assign_attributes school_info_params
+    puts "school info before existing sch--> #{school_info_params.inspect}"
     if existing_school_info.nil? || existing_school_info.changed?
-      submitted_school_info = SchoolInfo.where(new_school_info_params).
+      puts existing_school_info.changes
```

Comment:
@islemaster I add the puts statement the output shows that the value is not updated.  For instance, full_address is {(nil, "")}